crates/optimism/consensus/src/validation/isthmus.rs

```diff
@@ -0,0 +1,32 @@
+//! Block validation w.r.t. consensus rules new in isthmus hardfork.
```

Comment:
```suggestion
//! Block validation w.r.t. consensus rules new in Isthmus hardfork.
```
nit: for consistency